src/main/java/de/funfried/netbeans/plugins/external/formatter/ui/editor/ExternalFormatterReformatTaskFactory.java

```diff
@@ -56,6 +56,8 @@ public ReformatTask createTask(Context context) {
 
 		MimeType mimeType = MimeType.getMimeType(document);
 		if (mimeType != null) {
+			StatusDisplayer.getDefault().setStatusText("Document Mimetype = " + mimeType.getDisplayName());
```

Comment:
I guess that was not meant to be included into the commit, or?

Reply:
This was intentional because I'd love to see the applicable mimetype, e. g. when the SQL is just a text or property file.
However, this is not the hill I want to die on so I have reverted it.

Reply:
Did you not push yet? It's still there 🤔